README.md

```diff
@@ -7,9 +7,10 @@
 This repository has the source code for Comprehensive Rust 🦀, a multi-day Rust
 course developed by the Android team. The course covers all aspects of Rust,
 from basic syntax to generics and error handling. It also includes deep dives on
-[Android], [bare-metal], and [concurrency].
+[Android], [Chromium], [bare-metal], and [concurrency].
```

Comment:
Could you also

- add a line about the new deep dive here: https://google.github.io/comprehensive-rust/
- and a paragraph about it here: https://google.github.io/comprehensive-rust/running-the-course/course-structure.html

Feel free to merge this first and then add this right after if you want to do some more polishing of this new content.

Reply:
Thanks! There's already a line in the first list within this PR; 99e2d69 adds a paragraph to the latter.